components/panel/style.scss

```diff
@@ -134,6 +134,10 @@
 	&:empty {
 		margin-top: 0;
 	}
+
+	&:first-of-type {
+		margin-top: 0px;
```

Comment:
Minor: `0` doesn't need a unit. Further, this could be merged into the selector above:

```scss
&:empty,
&:first-of-type {
	margin-top: 0;
}
```